### DISCUSSION TITLE
Implement Attention through e3nn (version==0.1.0)

### DISCUSSION THREAD

**Discussion by Silencezong** [e3nn/e3nn#305] [Q&A]
Does anyone know how to use e3nn (0.1.0) to implement the Attention. Just use the Linear function in e3nn.linear？

**Comment by Silencezong** [comment]
Data includes Rs_in, Rs_out, and e3nn.point.data_helpers.DataNeighbors

**Reply by mariogeiger** [reply]
Here is a guide to create an attention layer with e3nn https://docs.e3nn.org/en/stable/guide/transformer.html
But it requires the last version (>=0.3.0)
